src/jit/codegenarmarch.cpp

```diff
@@ -1557,7 +1773,21 @@ void CodeGen::genCallInstruction(GenTreeCall* call)
             genConsumeReg(argNode);
             if (argNode->gtRegNum != argReg)
             {
-                inst_RV_RV(ins_Move_Extend(argNode->TypeGet(), argNode->InReg()), argReg, argNode->gtRegNum);
+#ifdef _TARGET_ARM_
+                if (curArgTabEntry->isSplit)
+                {
+                    for (unsigned idx = 0; idx < curArgTabEntry->numRegs; idx++)
+                    {
+                        regNumber regDst = (regNumber)((unsigned)argReg + idx);
+                        regNumber regSrc = (regNumber)((unsigned)argNode->gtRegNum + idx);
+                        inst_RV_RV(ins_Move_Extend(argNode->TypeGet(), argNode->InReg()), regDst, regSrc);
```

Comment:
`InReg()` is no longer used for the RyuJIT backend, in which the code generator should always know whether a node is in a register or not. I think you need `true` here.